<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowShuffleData</class>
 <widget class="QWidget" name="windowShuffleData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="labelAnalyzeResults">
     <property name="font">
      <font>
       <family>Avenir Next</family>
       <pointsize>20</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Analyze Results</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QToolButton" name="settingsButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="labelTitleShuffleData">
     <property name="text">
      <string>Shuffle ensembles</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="ButtonShuffling">
     <property name="text">
      <string>Start the shuffling</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>100</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="nextShuffleData">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
